airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/__init__.py

```diff
@@ -93,9 +105,15 @@
 SENSORS = [
```

Comment:
Just to check my understanding of these updated sensors:
1. when any metadata file changes, run `generate_registry_entry` to generate the registry entry for it
2. when any "latest" cloud registry entries change, run `generate_cloud_registry` to generate the registry for cloud
3. when any "latest" cloud registry entries change, run `generate_cloud_registry` to generate the registry for OSS

That right? One thing I'm not sure I understand is where the partitions come in and how they work. Is this to run the registry generation for both cloud/oss registries when metadata changes?

Reply:
Exactly right 

and as far as the partitions go

Partitions is the mechanism that lets us dynamically create an `asset` for a dynamic input.

In this case the dynamic input is a file (thats referenced by etag) so that we dont process a metadata file in to its registry entry files more than once as etag is unique to the file.

I will admit the way you define a partition, a partition job, a partion sensor and a partition asset(s) in dagger is not as straightforward as you would expect.